doc/UsersGuide/source/tables/fv3diagtable.csv

```diff
@@ -0,0 +1,416 @@
+nbdsf_ave,Near IR beam downward solar flux
+nddsf_ave,Near IR diffuse downward solar flux
+csulf_avetoa,Clear sky upward longwave flux at top of atmosphere
+csusf_avetoa,Clear sky upward shortwave flux at top of atmosphere
+csdlf_ave,Clear sky downward longwave flux
+csusf_ave,Clear sky upward shortwave flux
+csdsf_ave,Clear sky downward shortwave flux
+csulf_ave,Clear sky upward longwave flux
+DSWRFtoa,Top of atmosphere downward shortwave flux
+USWRFtoa,Top of atmosphere upward shortwave flux
+ULWRFtoa,Top of atmosphere upward longwave flux
+TCDC-aveclm,Atmosphere column total cloud cover
+TCDC_avebndcl,Boundary layer cloud layer total cloud cover
+TCDCcnvcl,Convective cloud layer total cloud cover
+PREScnvclt,Pressure at convective cloud top level
+PREScnvclb,Pressure at convective cloud bottom level
+TCDC_avehcl,High cloud level total cloud cover
+PRES_avehct,Pressure high cloud top level
+PRES_avehcb,Pressure high cloud bottom level
+TEMP_avehct,Temperature at high cloud top level
+TCDC_avemcl,Mid-cloud level total cloud cover
+PRES_avemct,Pressure at middle cloud top level
+PRES_avemcb,Pressure at middle cloud bottom level
+TEMP_avemct,Temperature middle cloud top level
+TCDC_avelcl,Low cloud level total cloud cover
+PRES_avelct,Pressure low cloud top level
+PRES_avelcb,Pressure at low cloud bottom level
+TEMP_avelct,Temperature at low cloud top level
+AOD_550,Total aerosol optical depth at 550 nm
+DU_AOD_550,Dust aerosol optical depth at 550 nm
+BC_AOD_550,Soot aerosol optical depth at 550 nm
+OC_AOD_550,Water soluble aerosol optical depth at 550 nm
+SU_AOD_550,Suspended organic aerosol optical depth at 550 nm
+SS_AOD_550,Salt aerosol optical depth at 550 nm
+aod,Total aerosol optical depth at 550 nm
+sw_upfxc,Total sky upward shortwave flux at top of atmosphere - GFS radiation
+sw_dnfxc,Total sky downward shortwave flux at top of atmosphere - GFS radiation
+sw_upfx0,Clear sky upward shortwave flux at top of atmosphere - GFS radiation
+lw_upfxc,Total sky upward longwave flux at top of atmosphere - GFS radiation
+lw_upfx0,Clear sky upward longwave flux at top of atmosphere - GFS radiation
+ssrun_acc,Accumulated surface storm water runoff
+evbs_ave,Direct evaporation from bare soil
+evcw_ave,Canopy water evaporation
+snohf,Snow phase change heat flux
+pah_ave,Total precipitation advected heat
+trans_ave,Transpiration
+sbsno_ave,Sublimation (evaporation from snow)
+snowc_ave,Snow cover - GFS lsm
+snowc,Snow cover
+soilm,Total column soil moisture content
+tmpmin2m,Min temperature at 2 meters (height)
+tmpmax2m,Max temperature at 2 meters (height)
+dusfc,Surface zonal momentum flux
+dvsfc,Surface meridional momentum flux
+shtfl_ave,Surface sensible heat flux
+lhtfl_ave,Surface latent heat flux
+totprcp_ave,Surface precipitation rate
+totprcpb_ave,Bucket surface precipitation rate
+gflux_ave,Surface ground heat flux
+dlwsfc,Time accumulated downward longwave flux at surface
+ulwsfc,Time accumulated upward longwave flux at surface

```

Comment:
```suggestion
dlwsfc,Time-accumulated downward longwave flux at surface
ulwsfc,Time-accumulated upward longwave flux at surface

```

Reply:
Did you mean to add the line break for this one and another below?

Reply:
Nope-fixed!